packages/pages/src/components/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface StandardProps {
+  yaTrack?: string;
```

Comment:
Is this actually used by the component anywhere? Should we remove it?

Reply:
It's extended by ImagePropsShared, I put it in a higher-level types file since I suspect other components might use it, as it seems to be a common control prop scheme for analytics settings.